tests/conftest.py

```diff
@@ -1591,6 +1591,57 @@ def additional_testsuite_properties(record_testsuite_property, pytestconfig):
     # add markers as separated property
     markers = ocsci_config.RUN["cli_params"].get("-m", "").replace(" ", "-")
     record_testsuite_property("rp_markers", markers)
+    from ocs_ci.utility.version import (
+        get_dr_hub_operator_version,
+        get_ocp_dr_cluster_operator_version,
+        get_odf_multicluster_orchestrator_version,
+        get_ocp_gitops_operator_version,
+        get_submariner_operator_version,
+        get_volsync_operator_version,
+    )
+    from ocs_ci.utility.utils import (
+        get_oadp_version,
+        get_acm_version,
+    )
```

Comment:
why its imported in the middle of function

Reply:
done